src/components/notifications/Notification.svelte

```diff
@@ -24,6 +26,10 @@
         />
       {:else if notification.type === 'error'}
         <XThinIcon color={$themeStore === 'light' ? '#ffd6d7' : '#fec3c3'} />
+      {:else if notification.type === 'info'}
```

Comment:
One thing I tend to do in situations like this where there are more than a couple cases in an `if/else` or `switch` statement though is create an object/map. It tends to be a little more performant than having to go through each case individually and I find them a little easier to read(though this is just a personal preference 😄).

What do you think of creating a map like:
```
  const iconMap = {
    info: {
      component: InfoThinIcon,
      props: {
        color: '#1e3a8a',
      },
    },
    error: {
      component: XThinIcon,
      props: {
        color: $themeStore === 'light' ? '#ffd6d7' : '#fec3c3',
      },
    },
    success: {
      component: CheckThinIcon,
      props: {
        color: $themeStore === 'light' ? '#b8ffd3' : '#002e12',
      },
    },
    warning: {
      component: WarningThinIcon,
      props: {
        color: '#7c2d12',
      },
    },
  }
```
then only needing to render a single component in the markup like:
```
<svelte:component this={iconMap[notification.type].component} {...iconMap[notification.type].props} />
```

Reply:
I like it! It does read more clearly when it's in a map like that. 💯 

Switched it over to use that.